build.gradle.kts

```diff
@@ -197,6 +197,8 @@ subprojects {
             }
 
             register<JvmTestSuite>("funTest") {
+                configurations["funTestImplementation"].extendsFrom(configurations["testImplementation"])
```

Comment:
Is this inheritance in terms of dependencies, e.g. funTestImpl inherits deps of testImpl?

Reply:
[Yes](https://docs.gradle.org/current/userguide/declaring_dependencies.html#sub:config-inheritance-composition).

Reply:
Hm, I was unaware of that inheritance. I believe that both, `´test` and `funTest` do have dependencies which should not be shared. 

Would it make sense to share code by having 3 projects like:

1. common-test-utils
2. test-utils
3. fun-test-utils

Reply:
I've found a yet better way to share (only the basic) dependencies across test suites, please have a look.